draft-ietf-rtcweb-jsep.xml

```diff
@@ -2436,8 +2442,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>The "a=rtcp-mux" line MUST only be added if present in
             the most recent answer.</t>
 
-            <t>The "a=rtcp-mux-only" line MUST only be added if present
-            in the most recent answer.</t>
+            <t>If the RTP/RTCP multiplexing policy is "require",
```

Comment:
Actually, according to your comment about -10, you can remove this para, since it basically just restates the earlier -only advice.